command/repo.go

```diff
@@ -508,7 +508,11 @@ func repoView(cmd *cobra.Command, args []string) error {
 				return fmt.Errorf("did not understand argument: %w", err)
 			}
 		} else {
-			toView = ghrepo.FromFullName(repoArg)
+			var err error
+			toView, err = ghrepo.FromFullName(repoArg)
+			if err != nil {
+				return fmt.Errorf("did not understand argument: %w", err)
```

Comment:
Nit: this will expand to a full error sentence that sounds like this: `did not understand argument: argument must be in format of OWNER/REPO`. It looks a bit repetitive.

How about `argument error: expected OWNER/REPO format, got "foo"`?

Reply:
Addressed in deaf453befc11023de5ab6c12f9c83dea97ea2e8